### PR TITLE
adding followings meta

### LINKED ISSUE #33: Followings

### PR DESCRIPTION
**Pull request by Fujistone**
Link to request : 
https://metabase.osp.dev/model/3175-suivis

closes #33

### CODE REVIEW

**Comment by Dynnammo** [comment]
Add meetings

File: followings/info.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+query:
+  model: true
+  sql: |
+    select decidim_follows.*, decidim_proposals_proposals.decidim_component_id
```
Comment (virgile-dev) [inline followings/info.yml:5]: add url

File: followings/info.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            join {{#users}} decidim_users on decidim_users.id = decidim_followable_id
+        where decidim_followable_type = 'Decidim::UserBaseEntity'
+  info:
+    meta:
+      depends_on:
+        - proposals
+        - components
+        - debates
+        - blogs_posts
+        - users
```
Comment (virgile-dev) [inline followings/info.yml:35]: missing
- budgets 
- initiative

File: followings/info.yml (hunk 1 above)
Comment (virgile-dev) [reply followings/info.yml:5]: and title

Comment (Dynnammo) [reply followings/info.yml:5]: Done

**Review by simonaszilinskas — CHANGES_REQUESTED** [review]
corriger le nombre de following - exemple 161
https://demo.decidim.opensourcepolitics.eu/processes/ep/f/115/proposals/161

**Comment by simonaszilinskas** [comment]
Rajouter le titre (followable_title)

**Comment by Dynnammo** [comment]
@simonaszilinskas just updated the request here : https://metabase.osp.cat/model/1885-suivis. We can review it whenever you want tomorrow

**Pull request merged**